packages/web/src/components/spa/HackAUBG/JourneySection/JourneySection.jsx

```diff
@@ -0,0 +1,13 @@
+import { JourneyDesktop } from './JourneyDesktop/JourneyDesktop';
+import { JourneyMobile } from './JourneyMobile/JourneyMobile';
+import { useMediaQuery } from 'react-responsive';
+
+export const JourneySection = ({ props }) => {
+    const isMobile = useMediaQuery({ query: '(max-width: 1200px)' });
```

Comment:
test whether it works with 1000px. This is the breakpoint we usually use. If it doesn't, leave it like that

Reply:
I know we use 1000 but the design with the lines and dots does not look ok on 1000 and i think it will be better if it renders the mobile one on 1200

Reply:
Ok, that's fine.